### PR TITLE
Monitoring section

### PR DESCRIPTION
**Pull request by kacperlukawski**
This PR introduces a separate section for monitoring, with a description of all the metrics. Those were taken from the codebase (master branch), so publishing after the next release might be better.

### CODE REVIEW

**Review by timvisee — APPROVED** [review]
Nice! I agree it is better to extract it to its own page.

Maybe we should do this on the `v1.2.x` branch though to incorporate it into the next release.

**Comment by kacperlukawski** [comment]
@timvisee Good point, thanks! I'll move those updated into `v1.2.x`

File: qdrant/v1.1.x/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+There are also some metrics which are exposed in distributed mode only.
+
+| Name                             | Type    | Meaning                                                                |
+|----------------------------------|---------|------------------------------------------------------------------------|
+| cluster_peers_total              | gauge   | Total number of cluster peers                                          |
+| cluster_term                     | counter | Current cluster term                                                   |
+| cluster_commit                   | counter | Index of last committed (finalized) operation cluster peer is aware of |
+| cluster_pending_operations_total | gauge   | Total number of pending operations for cluster peer                    |
+| cluster_voter                    | gauge   | Is cluster peer a voter or learner                                     |
```
Comment (davidmyriel) [inline qdrant/v1.1.x/monitoring.md:49]: Whether the cluster peer is a voter or learner

**Review by davidmyriel — APPROVED** [review]
lgtm, just 1 small change at the end

**Comment by kacperlukawski** [comment]
@timvisee @davidmyriel I changed the target to `v1.2.x` branch and made the requested changes

**Comment by kacperlukawski** [comment]
@timvisee Thanks for having a look! Done.

**Review by timvisee — APPROVED** [review]

**Review by agourlay — APPROVED** [review]

**Pull request merged**